tests/cli/test_cli.py

```diff
@@ -1771,6 +1776,24 @@ def test_run(ctx):
     )
     _run_command([cl, 'uedit', 'codalab', '--time-quota', ctx.time_quota])  # reset time quota
 
+    # Test that bundle fails when run without sufficient disk quota
+    _run_command([cl, 'uedit', 'codalab', '--disk-quota', f'{int(DISK_QUOTA_SLACK_BYTES)+1}'])
+    uuid = _run_command(
+        [cl, 'run', f'head -c {2*int(DISK_QUOTA_SLACK_BYTES)} /dev/zero > test.txt; sleep 100000',],
```

Comment:
Add a comment here explaining we're creating a file twice the size of DISK_QUOTA_SLACK_BYTES

Reply:
I took that out to make the necessary file size smaller and to make it less confusing. Now, the bundle requests 1MB and the running bundle creates a file of size 1MB + 10 bytes, thereby going over the disk quota and causing the kill request.

Does this look a bit better?

Reply:
You can see the change here: https://github.com/codalab/codalab-worksheets/pull/4307/files/d12b0512d4935427f657a47bfb921075e45c8da2..6a8b2825e2d353d3f99a1e747acdaa8fe82ca6c1